cmake/macos/resources/create-package.cmake.in

```diff
@@ -1,7 +1,35 @@
-set(CMAKE_PROJECT_NAME ${CMAKE_PROJECT_NAME})
-set(CMAKE_PROJECT_VERSION ${CMAKE_PROJECT_VERSION})
-set(MACOS_BUNDLEID ${MACOS_BUNDLEID})
-set(UUID_PACKAGE ${UUID_PACKAGE})
-set(UUID_INSTALLER ${UUID_INSTALLER})
-configure_file(cmake/macos/resources/installer-macos.pkgproj.in
-  "${CMAKE_CURRENT_BINARY_DIR}/installer-macos.generated.pkgproj")
+make_directory("$ENV{DESTDIR}${CMAKE_INSTALL_PREFIX}/package/Library/Application Support/obs-studio/plugins")
+
+if(EXISTS "$ENV{DESTDIR}${CMAKE_INSTALL_PREFIX}/@CMAKE_PROJECT_NAME@.plugin" AND NOT IS_SYMLINK "$ENV{DESTDIR}${CMAKE_INSTALL_PREFIX}/@CMAKE_PROJECT_NAME@.plugin")
+  file(INSTALL DESTINATION "$ENV{DESTDIR}${CMAKE_INSTALL_PREFIX}/package/Library/Application Support/obs-studio/plugins"
+    TYPE DIRECTORY FILES "$ENV{DESTDIR}${CMAKE_INSTALL_PREFIX}/@CMAKE_PROJECT_NAME@.plugin" USE_SOURCE_PERMISSIONS)
+
+  if(CMAKE_INSTALL_CONFIG_NAME MATCHES "^([Rr][Ee][Ll][Ee][Aa][Ss][Ee])$" OR CMAKE_INSTALL_CONFIG_NAME MATCHES "^([Mm][Ii][Nn][Ss][Ii][Zz][Ee][Rr][Ee][Ll])$")
```

Comment:
Would it be more readable to use a temp var to deal with case insensitive comparisons?
```cmake
string(TOLOWER "${CMAKE_INSTALL_CONFIG_NAME}" LOWER_CMAKE_INSTALL_CONFIG)
if(CMAKE_INSTALL_CONFIG_NAME MATCHES "^Release$" OR CMAKE_INSTALL_CONFIG_NAME MATCHES "^MinSizeRel$")
unset(LOWER_CMAKE_INSTALL_CONFIG)
```
or:
```cmake
string(TOLOWER "${CMAKE_INSTALL_CONFIG_NAME}" LOWER_CMAKE_INSTALL_CONFIG)
if(CMAKE_INSTALL_CONFIG_NAME STREQUAL "Release" OR CMAKE_INSTALL_CONFIG_NAME STREQUAL "MinSizeRel")
unset(LOWER_CMAKE_INSTALL_CONFIG)
```

No strong feelings either way.

Reply:
This follows the default code created by CMake for installation scripts (check the `cmake_install.cmake` script generated in every target directory).

Reply:
Unfortunate. Okay then.